vue-components/src/components/FeedbackMessage.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+	<div :class="[ 'wikit-FeedbackMessage', `wikit-FeedbackMessage--${type}` ]">
+		<Icon
+			class="wikit-FeedbackMessage__icon"
+			:type="type === 'error' ? 'error' : 'alert'"
+			:color="type"
+			size="medium"
+		/>
+		{{ message }}
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import Icon from './Icon.vue';
+
+export default Vue.extend( {
```

Comment:
You had a similar message for Input component saying it's internal and should not be used outside. Maybe we can have that in here too?

Reply:
Good point, thanks. Done!